legion/jupyterlab-plugin/jupyter_legion/handlers/cloud.py

```diff
@@ -0,0 +1,252 @@
+#
+#    Copyright 2019 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
```

Comment:
Missing the work dir parameter
![Screenshot from 2019-05-29 21-07-25](https://user-images.githubusercontent.com/6774206/58576678-f07bff80-8255-11e9-90f6-f95bcfbed1aa.png)

Reply:
Fixed